React/Modules/RCTUIManager.m

```diff
@@ -1520,6 +1521,8 @@ static void RCTMeasureLayout(RCTShadowView *view, RCTShadowView *ancestor, RCTRe
   moduleConstants[@"bubblingEventTypes"] = bubblingEventTypes;
   moduleConstants[@"directEventTypes"] = directEventTypes;
 
+  NSString *componentName = componentData.name;
```

Comment:
can we avoid dot notation please?

Reply:
The original used dot notation, but I guess we might as well switch to bracket notation if we're refactoring here.